### PR TITLE
Working version of Hello World

### PR DESCRIPTION
**Pull request by samarabbas**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,9 @@
 # Log file
 *.log
 
+# vscode files
+.vscode
```
Comment (HibbaAbas) [inline .gitignore:8]: Hmmmm... interesting...
Do you mind explaining why this is needed.

Comment (samarabbas) [reply .gitignore:8]: Hmmmm... Very interesting... I don't think this needs any explanation.  
Do you mind approving this PR as it is absolutely necessary for us to land it.

Comment (HibbaAbas) [reply .gitignore:8]: If you don't explain this u are fired maggot.

**Comment by samarabbas** [comment]
yayy!!!

**Pull request merged**